heuristic/heuristic_calculations.py

```diff
@@ -1,322 +1,211 @@
 from collections import defaultdict
 from operator import itemgetter
 
-# todo: convert to class and store data to reduce overhead?
-
-
-def calculate_deviation_from_demand(data, y):
+def calculate_deviation_from_demand(model, y):
```

Comment:
What is the purpose of this file now that we have `Alns_calculator`?

Reply:
ALNS_calculator should not have been included in the merge. That was my fault for not removing old files. It was my first thought that it could be a good idea, but I abandoned it for delta_calculations and heuristic calculations instead. It is outdated.